### PR TITLE
Feature/repeat wait

### PR DESCRIPTION
**Pull request by Tjerk258**
repeat and wait added to the code

### CODE REVIEW

File: Core/Src/logic_layer.c [hunk 1]
```diff
@@ -106,7 +180,7 @@ uint8_t r3g3b2_Colour(char command[])
  */
 int logic_layer(char commando[])
 {
-	char commando_list[NUMBER_OF_COMMANDS][MAX_NUMBER_OF_SCRIPT_CHARACTER] = {"lijn", "rechthoek", "tekst", "bitmap", "clearscherm", "cirkel", "figuur" }; // The script commando that the project needs to do.
+	char commando_list[NUMBER_OF_COMMANDS][MAX_NUMBER_OF_SCRIPT_CHARACTER] = {"lijn", "rechthoek", "tekst", "bitmap", "clearscherm", "cirkel", "figuur", "herhaal", "wait"}; // The script commando that the project needs to do.
```
Comment (Shariefie) [inline Core/Src/logic_layer.c]: "wait" should be "wacht", same as in script.

**Review by Shariefie — APPROVED** [review]
Your code looked good, didnt have much to say. Just a reminder to change the word wait to wacht as in script.

File: Core/Inc/main.h [hunk 2]
```diff
@@ -38,7 +37,7 @@ extern "C" {
 #include "stdio.h"
 #include "string.h"
 //#include "API.h"
-#include "logic_layer.h"
+//#include "logic_layer.h"
```
Comment (bestehustudentEU) [inline Core/Inc/main.h]: instead of commenting out can you not delete this line?

File: Core/Src/usart.c [hunk 3]
```diff
@@ -30,6 +30,7 @@ UART_HandleTypeDef huart2;
 
 void MX_USART2_UART_Init(void)
 {
+
```
Comment (bestehustudentEU) [inline Core/Src/usart.c:33]: please remove this empty lines

**Review by bestehustudentEU — COMMENTED** [review]
minor problems

File: Core/Src/logic_layer.c (hunk 1 above)
Comment (Tjerk258) [reply Core/Src/logic_layer.c]: changed wait to wacht

File: Core/Inc/main.h (hunk 2 above)
Comment (Tjerk258) [reply Core/Inc/main.h]: deleted

File: Core/Src/usart.c (hunk 3 above)
Comment (Tjerk258) [reply Core/Src/usart.c:33]: This empty line is created by code generation of stm32cubemx

File: Core/Src/main.c [hunk 4]
```diff
@@ -112,7 +115,13 @@ int main(void)
   while (1)
   {
 
-	  FL_Parser();
+//	  FL_Parser();
```
Comment (bestehustudentEU) [inline Core/Src/main.c]: weg halen!

Comment (Tjerk258) [reply Core/Src/main.c]: gifixed

**Review by bestehustudentEU — APPROVED** [review]

**Pull request merged**